docs/docs/identity-providers/gitlab.md

```diff
@@ -0,0 +1,48 @@
+---
+title: GitLab
+lang: en-US
+sidebarDepth: 0
+meta:
+  - name: keywords
+    content: gitlab oidc openid-connect identity-provider
+---
+
+# GitLab
+
+Log in to your Gitlab or create an account if you don't have one [here](https://gitlab.com/users/sign_in)
+
+Go to the user settings which can be found in the user profile to [create an application](https://gitlab.com/profile/applications) where you will get your app credentials
+
+![create an application](./img/gitlab/gitlab-create-applications.png)
+
+On the **Applications** page, add a new application by setting the following parameters:
+
+Field        | Description
+------------ | --------------------------------------------
+Name         | The name of your web app
+Redirect URI | `https://${redirect-url}/oauth2/callback`
```

Comment:
Redirected URI is an old configuration option.  The new value is `authenticate_service_url` .